tests/ui/cfg_attr_lint.rs

```diff
@@ -0,0 +1,17 @@
+#![feature(tool_lints)]
+
+#![warn(clippy::deprecated_cfg_attr)]
+
+// This doesn't get linted, see known problems
+#![cfg_attr(rustfmt, rustfmt_skip)]
+
+#[cfg_attr(rustfmt, rustfmt_skip)]
+fn main() {
+    foo::f();
+}
+
```

Comment:
I think this could use some more tests, just to be sure. rustfmt has a test file with some more examples, we can probably just take most of them: [rust-lang-nursery/rustfmt/tests/target/skip.rs](https://github.com/rust-lang-nursery/rustfmt/blob/1cca758d4d1ca0543b0f14eabbdcdd201f20f80b/tests/target/skip.rs)